test/unit/plugins/synced_folders/nfs/cap/mount_options_test.rb

```diff
@@ -0,0 +1,96 @@
+require_relative "../../../../base"
+
+require_relative "../../../../../../plugins/synced_folders/nfs/cap/mount_options"
+
+describe VagrantPlugins::SyncedFolderNFS::Cap::MountOptions do
+
+  let(:caps) do
+    VagrantPlugins::SyncedFolderNFS::Plugin
+      .components
+      .synced_folder_capabilities[:nfs]
+  end
+  let(:cap){ caps.get(:mount_options) }
+
+  let(:machine) { double("machine") }
+  let(:comm) { VagrantTests::DummyCommunicator::Communicator.new(machine) }
+  let(:mount_owner){ "vagrant" }
+  let(:mount_group){ "vagrant" }
+  let(:mount_uid){ "1000" }
+  let(:mount_gid){ "1000" }
+  let(:mount_name){ "vagrant" }
+  let(:mount_guest_path){ "/vagrant" }
+  let(:folder_options) do
+    {
+      owner: mount_owner,
+      group: mount_group,
+      hostpath: "/host/directory/path"
+    }
+  end
+
+  before do
+    allow(machine).to receive(:communicate).and_return(comm)
+    allow(machine).to receive_message_chain(:env, :host, :capability?).with(:smb_mount_options).and_return(false)
+    allow(ENV).to receive(:[]).with("VAGRANT_DISABLE_SMBMFSYMLINKS").and_return(true)
+    allow(ENV).to receive(:[]).with("GEM_SKIP").and_return(false)
```

Comment:
Can use the `stub_env` helper here:
 
```ruby
stub_env("GEM_SKIP" => nil)
```

with env vars string values or nil should be used to simulate expected values (since ENV values can only be String or nil types)